app/pathogen/arbovirus/dashboard/ArboStudyPopup.tsx

```diff
@@ -53,36 +71,27 @@ export default function ArboStudyPopup(record: any) {
             {/*Header section*/}
             <div className={"py-2 px-4"}>
                 <div className="text-lg font-bold">
-                   {`${record.pathogen} Estimate Study`}
+                   {`${record.pathogen} Estimate`}
                 </div>
                 <div className={"text-sm text-blue-600"}>
                     {record.url ? <a href={record.url} target="_blank" rel="noopener noreferrer"> {record.url} </a> : "NO URL"}
                 </div>
             </div>
             {/*SeroPrev section*/}
-            <div className={"flex justify-between bg-gray-200 w-full py-2 px-4"}>
-                <div className={"w-1/3 font-semibold"}>
-                    {"Seroprevalence"}: <b> {`${(record.seroprevalence * 100).toFixed(1)}%`}</b>
-                </div>
-                <div className={"w-2/3"}>
-                    {(record.sample_start_date && record.sample_end_date) && (
-                        <>
-                            {`${TranslateDate(record.sample_start_date)} to ${TranslateDate(record.sample_end_date)}`}
-                        </>)
-                    }
+            <div className={`flex justify-between bg-${record.pathogen.toLowerCase()} w-full py-2 px-4`}>
+                <div className={"font-semibold"}>
+                    {"Seroprevalence"}: <b> {`${(record.seroprevalence * 100).toFixed(1)}% (Arbovirus: ${pathogenFullString(record.pathogen)})`}</b>
                 </div>
             </div>
             {/*Content section*/}
             <div className={"py-2 px-4 max-h-[250px] overflow-auto"}>
+                {row("Date", `${TranslateDate(record.sample_start_date)} to ${TranslateDate(record.sample_end_date)}`)}
                 {row("Inclusion Criteria", record.inclusion_criteria ? record.inclusion_criteria : "Not Reported")}
                 {row("Location", getGeography(record.city, record.state, record.country))}
                 {row("Sample Size", record.sample_size?.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ","))}
                 {row("Antibody Target",  record.antibodies.join(', '))}
```

Comment:
Should I color Antibody target? It was not requested in the ticket but we could color it like how we do in the table.

Reply:
I think the modal has pretty minimal color and looks okay the way it is but if that's something you'd like to do I'm down.

Reply:
It is colored and spaced correctly!